python/semantic_kernel/kernel.py

```diff
@@ -443,7 +460,6 @@ def add_plugins(self, plugins: list[KernelPlugin] | dict[str, KernelPlugin]) ->
     def add_function(
```

Comment:
One general thought: in dotnet, they don't require names to be specified when creating a plugin or a function. If not specified, it generates a random name to use. In Python, right now, we are requiring those to be specified. Shouldn't we allow for these to be created in Python without a required name and then generate a random name at time of creation?